src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/commands.py

```diff
@@ -249,7 +54,292 @@ def load_command_table(self, _):
         g.command('keys renew', 'regenerate_key', transform=lambda x: getattr(x, 'keys', x))
         g.command('keys list', 'list_keys', transform=lambda x: getattr(x, 'keys', x))
 
-    #with self.command_group('storage account network-rule', storage_account_sdk, min_api='2017-06-01', resource_type=ResourceType.MGMT_STORAGE, storage_client_factory=storage_client_factory(self.cli_ctx).storage_accounts) as g:
-    #    g.custom_command('add', 'add_network_rule')
-    #    g.custom_command('remove', 'remove_network_rule')
-    #    g.custom_command('list', 'list_network_rules')
+    with self.command_group('storage account', cloud_data_plane_sdk) as g:
+        g.storage_command('generate-sas', 'generate_shared_access_signature', command_type=cloud_data_plane_sdk)
```

Comment:
I'm confused why you you needed to put this in a separate command_group statement and why you needed to specify `cloud_data_plane_sdk` in both the command group call and in the storage command call.  If nothing else, you should be able to include this in the regular `storage account` command group and override command_type with the cloud_data_plane_sdk.  If this doesn't work, I suspect it's a bug on my part.